src/Acme/Bundle/RangeBundle/Normalizer/Storage/Product/RangeNormalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Acme\Bundle\RangeBundle\Normalizer\Storage\Product;
+
+use Acme\Bundle\RangeBundle\Model\Range;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * @author    Damien Carcel (damien.carcel@akeneo.com)
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class RangeNormalizer implements NormalizerInterface
```

Comment:
You mean one standard normalizer that support both "standard" and "storage"? Didn't even think of it, I admit, I just kept it the more similar to our existing attribute types.